runtime/buffer.cc

```diff
@@ -246,6 +241,10 @@ int optimize_copy_dims(copy_dim* dims, int rank) {
 
 }  // namespace
 
+// Use a little macro helper here to make calls to alloca() less verbose
+#define ALLOC_COPY_DIMS_ON_STACK(RANK) \
```

Comment:
I think we should probably just have `SLINKY_STACK_ALLOC(T, N)` in `util.h` and use that here and in evaluate.cc, and I think there might be other places that use it too.